improver/calibration/reliability_calibration.py

```diff
@@ -216,24 +216,26 @@ def _check_forecast_consistency(self, forecasts):
                    'forecast periods found: {}.')
             raise ValueError(msg.format(n_cycle_hours, n_forecast_periods))
 
-    def _create_cycle_hour_coord(self, forecast_reference_time):
+    def _create_unified_frt_coord(self, forecast_reference_time):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5066530)